src/Plugin/Block/ContactsEntity.php

```diff
@@ -380,4 +419,55 @@ public function calculateDependencies() {
     return $dependencies;
   }
 
+  /**
+   * Get list of links to display on manage block.
+   *
+   * @return array
+   *   Array of links to be based to an 'item_list' render array.
+   */
+  protected function getManageLinks() {
+    $entity_id = $this->pluginDefinition['_entity_type_id'];
+    $bundle_id = $this->pluginDefinition['_bundle_id'];
+    $entity_definition = $this->entityTypeManager->getDefinition($entity_id);
+    $bundle_type = $entity_definition->getBundleEntityType();
+    $operations = [];
+    // Add manage fields and display links if this entity type is the bundle
+    // of another and that type has field UI enabled.
+    if ($bundle_type && $entity_definition->get('field_ui_base_route')) {
+      $account = $this->currentUser;
```

Comment:
No need to assign to a variable as it's not a getter, just use `$this->currentUser->hasPermission()`